src/components/CellarStats.tsx

```diff
@@ -14,34 +14,35 @@ import { CurrentDeposits } from "./CurrentDeposits"
 import { InformationIcon } from "./_icons"
 import { analytics } from "utils/analytics"
 import { debounce } from "lodash"
-import { Apy } from "./Apy"
 import { isCurrentDepositsEnabled } from "data/uiConfig"
 import { ConfigProps } from "data/types"
 
 interface CellarStatsProps extends StackProps {
-  tvm?: ReactNode
-  apy?: ReactNode
-  apyTooltip?: string
+  firstTooltip?: string
```

Comment:
Can we have a better naming instead using "first" and "second"

Reply:
do you have better suggestion? 
I used that because we don't use spesific value for that component

Reply:
or maybe should I just create seperate component?